app/controllers/admin/courts_controller.rb

```diff
@@ -129,5 +131,13 @@ def area_of_law_by_id_or_name
       return area_of_law if area_of_law.present?
       AreaOfLaw.where(name: AreaOfLaw::Name::CHILDREN).first
     end
+
+    def process_update_exception(e)
+      if e.message.include?('postcode_court')
```

Comment:
Im not sure what is happening in this code - is it that you are converting a 'postcode_court' validation error into a 'postcode' validation error ?  If so, why is this not being done in the model along with other validation code ?